src/js/utils/form-utils.jsx

```diff
@@ -64,16 +64,23 @@ export const renderFormFields = ({
           typeof FieldLabel === 'string' ?
             <label htmlFor={attr.id} className="col-md-2 col-7 col-form-label col-form-label-xs text-center text-md-right">
               {FieldLabel && <Translate id={FieldLabel} defaultMessage={defaultMessage} />}
-              { attr.withTooltip ?
+              {attr.withTooltip &&
                 <Tooltip
-                  title={translate(tooltip, tooltip)}
+                  interactive="true"
+                  arrow="true"
+                  trigger={trigger}
+                  hideOnClick="true"
+                  html={tooltipLinks
+                    ? ReactHtmlParser(translate(tooltip, tooltip, tooltipLinks))
```

Comment:
This `tooltipLinks` parameter should be named more generic because it will not always be the link.